ihatemoney/forms.py

```diff
@@ -9,9 +9,23 @@
 from flask_wtf.form import FlaskForm
 from markupsafe import Markup
 from werkzeug.security import check_password_hash, generate_password_hash
-from wtforms.fields.core import Label, SelectField, SelectMultipleField
-from wtforms.fields.html5 import DateField, DecimalField, URLField
-from wtforms.fields.simple import BooleanField, PasswordField, StringField, SubmitField
+from wtforms.fields import (
+    BooleanField,
+    DateField,
+    DecimalField,
+    Label,
+    PasswordField,
+    SelectField,
+    SelectMultipleField,
+    StringField,
+    SubmitField,
+)
+
+try:
+    # Compat for WTForms <= 2.3.3
```

Comment:
Why should we keep backwards compatibility?

Reply:
That's exactly my question. My experience showed me that jumping early on the "new backward incompatible release" train can be really annoying. So I tend to prefer solutions that introduce a backward compatibility layer, but I think this is not possible here, due to the default change for string values.

Reply:
> My experience showed me that jumping early on the "new backward incompatible release" train can be really annoying.

Ah, that seems bad indeed. Could you explain why?